cumulusci/tasks/salesforce/org_settings.py

```diff
@@ -74,33 +68,106 @@ def capitalize(s):
 
 
 @contextlib.contextmanager
-def build_settings_package(settings: dict, api_version: str):
+def build_settings_package(
+    settings: Optional[dict], object_settings: Optional[dict], api_version: str
+):
     with temporary_dir() as path:
-        os.mkdir("settings")
-        for section, section_settings in settings.items():
-            settings_name = capitalize(section)
-            if section == "orgPreferenceSettings":
-                values = "    " + "\n    ".join(
-                    line
-                    for line in "\n".join(
-                        ORGPREF.format(name=capitalize(k), value=v)
-                        for k, v in section_settings.items()
-                    ).splitlines()
+        if settings:
```

Comment:
Not necessary at all, but it seems like this method could be refactored to call two methods internally with prototypes: `_generate_settings_file(settings: Optional[dict])` and `_generate_object_settings_file(object_settings: Optional[dict])`. Then after those methods, you would have what's currently on lines 111-115.

The first line of each of those methods would be to check for the presence of the argument, and if the value is `None`, then they would just return. This would also help mitigate the multiple levels of indentation occurring.

Again, not necessary at all, I'm just partial to smaller functions myself.